### PR TITLE
Use ContextCompat.getSystemService().

### PR DESCRIPTION
**Pull request by Isira-Seneviratne**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -29,6 +29,9 @@ android {
         sourceCompatibility = JAVA_VERSION
         targetCompatibility = JAVA_VERSION
     }
+    kotlinOptions {
+        jvmTarget = JAVA_VERSION
```
Comment (dbrant) [inline app/build.gradle:33]: I don't believe we need any changes to `build.gradle` to use `ContextCompat.getSystemService()`, do we?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 1 above)
Comment (Isira-Seneviratne) [reply app/build.gradle:33]: That was for the `getSystemService` extension function, which inlines to `ContextCompat.getSystemService()`.

Comment (dbrant) [reply app/build.gradle:33]: Can we remove it?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/ktx/Context.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@file:JvmName("ContextUtil")
+
+package org.wikipedia.ktx
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+import androidx.core.content.getSystemService
+
+fun Context.setPlainText(label: CharSequence?, text: CharSequence?) {
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/ktx/Context.kt:10]: `setPlainText()` does not make sense as an extension function of `Context`.  Can we keep the file structure the way it was?

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/ktx/Context.kt (hunk 2 above)
Comment (Isira-Seneviratne) [reply app/src/main/java/org/wikipedia/ktx/Context.kt:10]: Sure.

**Pull request closed without merging**